Author: Daniel Kobras <kobras@debian.org>
Description: Removing inclusion of config.h from dv_types.h revealed two
 files that should include it themselves.
---
 encodedv/encodedv.c     |    4 ++++
 encodedv/insert_audio.c |    4 ++++
 2 files changed, 8 insertions(+)

--- libdv.orig/encodedv/encodedv.c
+++ libdv/encodedv/encodedv.c
@@ -24,6 +24,10 @@
  *  The libdv homepage is http://libdv.sourceforge.net/.  
  */
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
 #include <string.h>
 #include "libdv/dv_types.h"
 #include "libdv/dv.h"
--- libdv.orig/encodedv/insert_audio.c
+++ libdv/encodedv/insert_audio.c
@@ -23,6 +23,10 @@
  *  The libdv homepage is http://libdv.sourceforge.net/.  
  */
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
 #include "libdv/dv_types.h"
 
 #include <stdio.h>
